docs/testplan/FEC_test.md

```diff
@@ -0,0 +1,29 @@
+# SONiC Switch FEC Sanity Test
+
+- [SONiC Switch FEC Sanity Test](#sonic-switch-fec-sanity-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Background](#background)
+  - [Test Steps](#test-steps)
+
+## Test Objective
+
+This test aims to verify that SONiC switch links properly respond to severe FEC errors by shutting down the operational status of affected ports.
```

Comment:
@sm-xu why do you want to shutdown? The PCS has high BER detection mandated by IEEE where if the BER is above a threshold, the PCS shutdown and keeps the link disabled. This happens in hardware

![image](https://github.com/user-attachments/assets/bc88bfcf-936b-4705-9138-316a9a2109cb)